app/views/developers/index.html.erb

```diff
@@ -1,13 +1,114 @@
-<div class="banner">
-  <%= link_to "I'm a Developer", companies_path, style: "color: red"  %>
-  <%= link_to "I'm a Startup", developers_path%>
-</div>
-
-
-<h1>All developers</h1>
-
-<ul>
-<% @developers.each do |developer| %>
-  <li><%= link_to developer.first_name, developer_path(developer) %></li>
- <% end %>
-</ul>
+<div class="banner" style="background-image: linear-gradient(rgba(0,0,0,0.6),rgba(0,0,0,0.6)), url(https://images.unsplash.com/photo-1522202176988-66273c2fd55f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1051&q=80);">
+  <div class="customer-segment-selector">
+    <%= link_to "I'm a Developer", companies_path, class: "btn banner-button developer-button" %>
+    <%= link_to "I'm a Startup", developers_path, class: "btn banner-button startup-button-current" %>
+  </div>
+  <div class="banner-text">
+    <h1>Do You Want to Find Developers Who Share Your Startups Values?</h1>
+    <p>Find Your Developer Now!</p>
+    <div class="banner-search-bar">
+      <input class="form-control" type="text" placeholder="Search" aria-label="Search">
+      <a class="btn btn-flat" href="#">Search</a>
+    </div>
+  </div>
+</div>
+<div class="container">
+  <div class="unique-value-proposition">
+    <h1>Supporting Your Mission With Like-Minded Developers</h1>
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rem reiciendis corporis officia aut, fuga cumque odio cupiditate minus voluptatem consectetur, quam mollitia eum suscipit, sunt quas magni vel ab commodi!</p>
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugiat incidunt nemo in quasi praesentium est ratione officiis sed unde, excepturi sit, rerum id aperiam consequatur commodi reiciendis aliquam quis veritatis.</p>
+  </div>
+</div>
+<div class="green-carousel">
+  <div class="container">
+    <div class="carousel-title">
+      <h1>Find a developer in your city:</h1>
+    </div>
+    <div class="carousel">
+      <% @developers.sample(10).each do |developer| %>
```

Comment:
should we only show developers that are currently available?

Reply:
do think we should address this but will merge anyways as we should have this in master. address on another branch maybe?